src/include/timestamp_inline.h

```diff
@@ -200,6 +200,14 @@
             (dest)->prepare = 1;                                                              \
     } while (0)
 
+/* Return true if the time aggregates are the same. */
+#define WT_TIME_AGGRTEGATES_EQUAL(ta1, ta2)                                                       \
```

Comment:
There is a spelling error here "AGGREGATES", I also think it shouldn't be plural (similar to `WT_TIME_AGGREGATE_MERGE`). I'd choose `WT_TIME_AGGREGATE_EQUAL`.

Could this memcmp the two data structures instead? That would be more robust to changes in the fields.

Reply:
Fixed the spelling error and changed it to use `memcmp`.

Reply:
Reverted the use of `memcmp` usage due to the padding in structure size.